src/tests/all-tests.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+var fluid = fluid || require('infusion');
+var gpii  = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.test.mail.smtp");
+require("../js/mailserver");
+
+var jqUnit = fluid.require("jqUnit");
+var fs     = require("fs");
+
+var nodemailer    = require("nodemailer");
+var smtpTransport = require('nodemailer-smtp-transport');
+var options = {
```

Comment:
Many of these moved around based on other comments, the rest are definitely namespaced.